.github/workflows/ci.yml

```diff
@@ -20,61 +20,61 @@ jobs:
-      run: bash $GITHUB_WORKSPACE/.github/scripts/test-retry.sh yarn test-clients
-    - name: yarn test-docs
-      run: yarn test-docs
-    - name: yarn test-treemap
-      run: bash $GITHUB_WORKSPACE/.github/scripts/test-retry.sh yarn test-treemap
+      - name: yarn test-clients
+        run: bash $GITHUB_WORKSPACE/.github/scripts/test-retry.sh yarn test-clients
+      - name: yarn test-docs
+        run: yarn test-docs
+      - name: yarn test-treemap
+        run: bash $GITHUB_WORKSPACE/.github/scripts/test-retry.sh yarn test-treemap
 
-    - run: yarn diff:sample-json
-    - run: yarn diff:flow-sample-json
-    - run: yarn lint
-    - run: yarn test-lantern
-    - run: yarn test-legacy-javascript
-    - run: yarn i18n:checks
-    - run: yarn dogfood-lhci
+      - run: yarn diff:sample-json
+      - run: yarn diff:flow-sample-json
+      - run: yarn lint
+      - run: yarn test-lantern
+      - run: yarn test-legacy-javascript
+      - run: yarn i18n:checks
+      - run: yarn dogfood-lhci
 
-    # Fail if any changes were written to any source files or generated untracked files (ex, from: build/build-cdt-lib.js).
-    - run: git add -A && git diff --cached --exit-code
+      # Fail if any changes were written to any source files or generated untracked files (ex, from: build/build-cdt-lib.js).
+      - run: git add -A && git diff --cached --exit-code
 
-    # buildtracker needs history and a common merge commit.
-    - name: Fixup git history (for buildtracker)
-      run: bash $GITHUB_WORKSPACE/.github/scripts/git-get-shared-history.sh
-      env:
-        # https://buildtracker.dev/docs/guides/github-actions#configuration
-        BT_API_AUTH_TOKEN: ${{ secrets.BT_API_AUTH_TOKEN }}
-    - name: Store in buildtracker
-      # TODO(paulirish): Don't allow this to fail the build. https://github.com/paularmstrong/build-tracker/issues/200
-      run: yarn bt-cli upload-build || true
-      env:
-        # https://buildtracker.dev/docs/guides/github-actions#configuration
-        BT_API_AUTH_TOKEN: ${{ secrets.BT_API_AUTH_TOKEN }}
+      # buildtracker needs history and a common merge commit.
+      - name: Fixup git history (for buildtracker)
+        run: bash $GITHUB_WORKSPACE/.github/scripts/git-get-shared-history.sh
+        env:
+          # https://buildtracker.dev/docs/guides/github-actions#configuration
+          BT_API_AUTH_TOKEN: ${{ secrets.BT_API_AUTH_TOKEN }}
+      - name: Store in buildtracker
+        # TODO(paulirish): Don't allow this to fail the build. https://github.com/paularmstrong/build-tracker/issues/200
+        run: yarn bt-cli upload-build || true
+        env:
+          # https://buildtracker.dev/docs/guides/github-actions#configuration
+          BT_API_AUTH_TOKEN: ${{ secrets.BT_API_AUTH_TOKEN }}
 
-    - name: Upload dist
-      uses: actions/upload-artifact@v1
-      with:
-        name: dist
-        path: dist/
+      - name: Upload dist
+        uses: actions/upload-artifact@v4
```

Comment:
This was [`using a version of actions/upload-artifact that is scheduled for deprecation`](https://github.com/GoogleChrome/lighthouse/actions/runs/9568114661), so I updated it to the latest major version.

Reply:
You're welcome, and sure; I removed the changes from formatting on save.